java/java.source.base/src/org/netbeans/modules/java/source/builder/TreeFactory.java

```diff
@@ -1914,19 +1918,36 @@ public LinkTree LinkPlain(ReferenceTree ref, List<? extends DocTree> label) {
     }
 
     public ReferenceTree Reference(ExpressionTree qualExpr, CharSequence member, List<? extends Tree> paramTypes) {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
Not sure if this is a blocker for merging this.

Reply:
It is just a code-style suggestion :)

Reply:
Please note this PR doesn't change the convention. All methods in the `TreeFactory` start with capital leter.